api/src/main/java/org/openmrs/module/fhir2/providers/r3/PractitionerFhirResourceProvider.java

```diff
@@ -60,6 +70,35 @@ public Practitioner getPractitionerById(@IdParam @NotNull IdType id) {
 		return Practitioner30_40.convertPractitioner(practitioner);
 	}
 	
+	@Create
+	public MethodOutcome createPractitioner(@ResourceParam Practitioner practitioner) {
+		return FhirProviderUtils.buildCreate(practitionerService.create(Practitioner30_40.convertPractitioner(practitioner)));
+	} 
+	
+	@Update
+	@SuppressWarnings("unused")
+	public MethodOutcome updatePractitioner(@IdParam IdType id, @ResourceParam Practitioner practitioner) {
+		if (id == null || id.getIdPart() == null) {
+			throw new InvalidRequestException("id must be specified to update");
+		}
+
+		practitioner.setId(id.getIdPart());
+
+		return FhirProviderUtils.buildUpdate(practitionerService.update(id.getIdPart(),
+				Practitioner30_40.convertPractitioner(practitioner)));
+	}
+	
+	@Delete
+	@SuppressWarnings("unused")
+	public OperationOutcome deletePractitioner(@IdParam @NotNull IdType id) {
+		org.hl7.fhir.r4.model.Practitioner practitioner = practitionerService.delete(id.getIdPart());
+		if (practitioner == null) {
+			throw new ResourceNotFoundException("Could not find practitioner to delete with id " + id.getIdPart());
+		}
+
+		return OperationOutcome30_40.convertOperationOutcome(FhirProviderUtils.buildDelete(practitioner));
```

Comment:
please rewrite this as `FhirProviderUtils.buildDelete(Practitioner30_40.convertPractitioner(practitioner));`